### DISCUSSION TITLE
礼貌交流

### DISCUSSION THREAD

**Discussion by mengchaoheng** [mengchaoheng/SCUT_thesis#47] [General]
最近因为我的不礼貌和@yzy1228682367在网上吵了起来，[原话题](https://github.com/MuiseDestiny/zotero-file/issues/47)已经不能回复，后续@yzy1228682367又在[jasminum](https://github.com/l0o0/jasminum/issues/200#issuecomment-1712400399)引用我以前[疏忽之处](https://github.com/zcchenvy/CIL-DDQN/issues/2)批评我。我开始是费解，然后是有点气愤，最后是发现自己确实不应该。于是借这个地方。我想起来以前在此项目下也和人吵过（见过大家争论）。礼貌也许确实是很重要的事。在此反思，并希望以后大家讨论都要有礼貌。

在此也再次重申对@yzy1228682367  的歉意：

感谢您之前分享教程给我，我看完教程有了自己想法并保留我的想法，但无论如何感谢你。如果我的话让你感到生气，我为此表示抱歉。

**Comment by mengchaoheng** [comment]
俗话说知耻而后勇，我在网上不够礼貌。以后一定引以为戒！！！

**Comment by mengchaoheng** [comment]
最近改论文改得有点暴躁，审论文也很暴躁，看到别人写的不好就想批评。冷静下来想想，都不容易。再次反思，放平心态。

**Reply by note286** [reply]
有时候我们看见一些人做出一些行为，会觉得不可思议，例如那种稍微仔细一点都不会犯的错误。但是很多时候我们一眼能看出问题来，是由于我们对此很熟悉，所以一般不会犯这种错误。但是每个人都有不擅长的地方，我们也会在其他人很擅长的地方犯很低级的错误。

不过，确实也存在一部分人，就是懒，不想付出就想要结果。对于这种人，也没必要废话什么，有时间和他交流，不如看看剧，望望景。

总之，世界很大，什么样的人都有，什么样的事情也都会发生。遇到我们擅长的事情，花一两分钟顺手指点一下就够了。如果闲的无聊，或者就有乐于助人的爱好，多花点时间也不是不行。这已经足够了，有时候我们的一句话，就会让网线对面的陌生人豁然开朗，也是一件善事。我们的能力和精力也有限，需要帮助的人也很多，我们不可能帮助所有人。如果觉得浪费时间和精力了，视而不见也没什么。

网络上交流大多时候都是文字，为了不必要的误解，多写一点，冗余一点，能避免很多问题。大家都说“放下助人情结，尊重他人命运”，也不必这么极端，顺手指点一下，就够了。绝大多数人也没什么说教的资格，真到了有说教的资格，你也不会去说教别人了。遇到问题，稍微指点一二就够了。都是讨论技术，忘记背后的人吧。毕竟不是在微博讨论关于人。还是仅关注技术本身就好。

随便说了些，有些相关，有些不相关。

**Reply by mengchaoheng** [reply]
特别感谢您的回复，我受益匪浅。读书更像是一场修行，渡人谈不上，渡己很重要。之所以写下这个帖子，除了提到的”导火索“，主要还是自己最近遇到一些事，有了许多感悟，静下心来发现戾气在心中滋生，把小事搞成大事，差点忘了自己最初的模样。我会继续不忘初心，与人为善，做好自己的工作，并能为自己帮助到其他人而感到快乐，这才是我所追求的最纯粹的人生。
